AppCenterDistribute/AppCenterDistribute/MSDistribute.m

```diff
@@ -62,10 +62,26 @@
 @implementation MSDistribute
 
 @synthesize channelUnitConfiguration = _channelUnitConfiguration;
+
 @synthesize updateTrack = _updateTrack;
 
 #pragma mark - Service initialization
 
++ (void)load {
+  [MSAppCenterUserDefaults addKeysToMigrate:@{
+    @"kMSDistributeIsEnabledKey" : @"MSAppCenterDistributeIsEnabledKey",              // MSDistributePrivate
+    @"MSPostponedTimestamp" : @"MSAppCenterPostponedTimestamp",                       // MSDistributePrivate
+    @"MSSDKHasLaunchedWithDistribute" : @"MSAppCenterSDKHasLaunchedWithDistribute",   // MSDistributePrivate
+    @"MSMandatoryRelease" : @"MSAppCenterMandatoryRelease",                           // MSDistributePrivate
+    @"MSDistributionGroupId" : @"MSAppCenterDistributionGroupId",                     // MSDistributePrivate
+    @"MSUpdateSetupFailedPackageHash" : @"MSAppCenterUpdateSetupFailedPackageHash",   // MSDistributePrivate
+    @"MSDownloadedReleaseHash" : @"MSAppCenterDownloadedReleaseHash",                 // MSDistributePrivate
+    @"MSDownloadedReleaseId" : @"MSAppCenterDownloadedReleaseId",                     // MSDistributePrivate
+    @"MSDownloadedDistributionGroupId" : @"MSAppCenterDownloadedDistributionGroupId", // MSDistributePrivate
+    @"MSTesterAppUpdateSetupFailed" : @"MSAppCenterTesterAppUpdateSetupFailed"        // MSDistributePrivate
```

Comment:
`// MSDistributePrivate` doesn't not really explain where it's used (it was a main intention for this).
I guess it makes sense to say that "DistributeIsEnabledKey" (for example) is used in `// [MSDistribute isEnabled]`